### DISCUSSION TITLE
Mikktspace support

### DISCUSSION THREAD

**Discussion by Bosa-pro** [assimp/assimp#4317] [Q&A] [answered]
Hi !
I apologize in advance - could not find another place to ask a question.
Does ASSIMP support mikktspace algoritm in tangent/bitangent calculation?
Thank you !

**Comment by kimkulling** [comment, marked as answer]
This feature is available as a draft. You can find it in the PR section. Feedback is really welcome. See https://github.com/assimp/assimp/pull/6265

I understand this feature has been pending for more than six months. I have restarted working on it.

**Reply by Bosa-pro** [reply]
Thanks!
